File: Assets/Omniscient Assets/Levels/Level Select Screen/Assets_By_Unluck_Activeden/Scripts/NewLvlSelect.js
#pragma strict
var scene = "";
function Start () {

}

function Update () {

	if (Input.GetMouseButtonDown(0)) // check for left-mouse
	{
    	var ray = Camera.main.ScreenPointToRay (Input.mousePosition);
    	var hit : RaycastHit;
    	if (collider && collider.Raycast (ray, hit, 100.0))
    	{
    		if (collider.tag == "LevelLight")
    		{
    			Application.LoadLevel(scene);
     	  	}
    	}
}
}

File: Assets/Omniscient Assets/Levels/Level Select Screen/Assets_By_Unluck_Activeden/Scripts/NewLvlSelect.cs
using UnityEngine;
using System.Collections;

public class NewLvlSelect : MonoBehaviour
{
	public string scene = "SP-Egypt-AR";
	
	public void Start()
	{
	}
	
	public void Update()
	{
		if (Input.GetMouseButtonDown(0))
		{
			Ray ray = Camera.main.ScreenPointToRay(Input.mousePosition);
			if (Physics.Raycast(ray, 100.0f))
			{
				GameState.SetCurrentLevel((string)scene);
				Application.LoadLevel("WeaponSwitcher");
			}
		}
	}
}

